PelotonEppSdk/Models/VariableType.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace PelotonEppSdk.Models
+{
+    public enum VariableType
+    {
+        String = 0
+    }
+}
```

Comment:
There is a directory for Enum PelotonEppSdk.Enums. Please move this enum there too.